client/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```

Comment:
Remove this function and the `reportWebVitals` file 
```suggestion
```

Reply:
done